README.md

```diff
@@ -71,13 +71,22 @@ The following transport options are available:
 
 ### `debug`
 
-#### Options
-
 A development option to write the html content of the email to a file for inspection.
 
+`transport: 'debug'`
+
+#### `debug options`
+
 - `dir` <String>: The location to save html to. Default: `./.emails`. This directory will be created if it does not exist.
 - `open` <Boolean>: If set to true, will automatically open the created html file in a browser.
 
+```
+transport: 'debug'
+transportOptions: {
+  dir: './emails',
+  open: true
+}
+```
```

Comment:
Please can you write this so that it follows the same format as the options for other transports, such as...

- `transport`:  'debug'
-  `transportOptions`:
    - `dir`: './path_to_dir'
    - `open`: true

Reply:
It was meant to be an example of how to use it like  https://github.com/UKHomeOfficeForms/hof-emailer#usage

I could title it usage if that helps?

Reply:
ok, fair enough, could you just give it a title of 'Example"